.github/workflows/gh-pages.yml

```
@@ -11,17 +11,17 @@ jobs:
     steps:
       - uses: actions/checkout@v2
         with:
-          submodules: true  # Fetch Hugo themes (true OR recursive)
-          fetch-depth: 0    # Fetch all history for .GitInfo and .Lastmod
+          submodules: recursive  # Fetch Hugo themes (true OR recursive)
+          fetch-depth: 1         # Fetch all history for .GitInfo and .Lastmod
```

```suggestion
          fetch-depth: 1         # Fetch all history for .GitInfo and .Lastmod
```
the comment is wrong now :)